compiler/crates/relay-lsp/src/server/lsp_state.rs

```diff
@@ -521,12 +530,34 @@ impl<TPerfLogger: PerfLogger + 'static, TSchemaDocumentation: SchemaDocumentatio
             js_server.process_js_source(uri, text);
         }
```

Comment:
Should this block live in the FileGroup::Source match case, since it's js-specific?